[2023-09-13T07:38:43.544+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-01T00:00:00+00:00 [queued]>
[2023-09-13T07:38:43.569+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-01T00:00:00+00:00 [queued]>
[2023-09-13T07:38:43.570+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:43.679+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-01 00:00:00+00:00
[2023-09-13T07:38:43.685+0000] {standard_task_runner.py:57} INFO - Started process 132 to run task
[2023-09-13T07:38:43.698+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-01T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp5ue46v4e']
[2023-09-13T07:38:43.705+0000] {standard_task_runner.py:85} INFO - Job 31: Subtask extract_data
[2023-09-13T07:38:43.814+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-01T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:44.009+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-01T00:00:00+00:00'
[2023-09-13T07:38:44.505+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:44.537+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230901T000000, start_date=20230913T073843, end_date=20230913T073844
[2023-09-13T07:38:44.648+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:44.690+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:18.423+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-01T00:00:00+00:00 [queued]>
[2023-09-15T11:31:18.450+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-01T00:00:00+00:00 [queued]>
[2023-09-15T11:31:18.453+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:18.479+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-01 00:00:00+00:00
[2023-09-15T11:31:18.496+0000] {standard_task_runner.py:57} INFO - Started process 362 to run task
[2023-09-15T11:31:18.529+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-01T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp9261qadi']
[2023-09-15T11:31:18.531+0000] {standard_task_runner.py:85} INFO - Job 22: Subtask extract_data
[2023-09-15T11:31:18.614+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-01T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:18.806+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-01T00:00:00+00:00'
[2023-09-15T11:31:19.139+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:19.167+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230901T000000, start_date=20230915T113118, end_date=20230915T113119
[2023-09-15T11:31:19.251+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:19.365+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
